cloud/kubernetes/helm/yugabyte/templates/service.yaml

```diff
@@ -185,6 +185,7 @@ spec:
           - "--master_addresses={{range $index := until (int ($root.Values.replicas.master))}}{{if ne $index 0}},{{end}}yb-master-{{ $index }}.yb-masters.$(NAMESPACE):7100{{end}}"
           - "--master_replication_factor={{ $root.Values.replicas.master }}"
           - "--metric_node_name=$(HOSTNAME)"
+          - "--memory_limit_hard_bytes={{ $root.Values.resource.master.limits.memory | regexFind "\\d+" | mul 1024 | mul 1024 | mul 870 }}"
```

Comment:
if you follow the template function route, you can just replace this with
`- "--memory_limit_hard_bytes={{ template "yugabyte.memory_hard_limit" $root.Values.resource.master }}"`

Reply:
Done!